capability_test.go

```diff
@@ -117,19 +117,19 @@ func TestReleasedClient(t *testing.T) {
 func TestNullClient(t *testing.T) {
```

Comment:
I think we can/should run this test in parallel.  Since the actual tests are happening in the context of a `t.Run`, we should also call `t.Helper()` for the top-level test.

Reply:
It seems like that probably applies to most if not all of the tests in this file, and this isn't something I added -- so do we maybe instead want to do a separate PR where we just go through and add t.Parallel() where appropriate?

Reply:
SGTM